.github/CONTRIBUTING.md

```diff
@@ -88,6 +88,37 @@ Provide as much detail as you can in each section to help us triage and process
 -   Feature suggestion - A missing feature you would like to see implemented in the project
 -   Other - Any other type of task related to the project
 
+### Definition of Done
+
+- Addresses open issue
+  - Meets listed acceptance criteria
+  - Key stakeholder approves (original issue author, product / business / QA / UX)
+- **Documentation** plays an important role in an open-source library project such as PWA Studio.
+
+  The following changes require documentation:
+
+  - **Public API** -
+    API documentation provide reference information and examples to help developers understand how to use the PWA Studio library.
+    Most of this documentation come from JSDoc blocks in the source code, so
+    changes that add or modify the public API also require JSDoc block changes.
+
+  - **New project concepts** -
+    Conceptual topics provide background knowledge to help developers understand how the different technologies come together.
+    If a change or feature requires a developer to learn a new pattern, design, or process, then that information needs to be documented.
+    Examples of project concepts include: UPWARD, scaffolding, and project configuration.
+  
+  - **Code functionality** -
+    Documentation is required when major changes happen in the project codebase that affect how the code works.
+    Communicating these changes help maintainers and early adopters keep up with the project.
+    These docs often take the form of release notes.
+
+- Test coverage:
+  - Unit tests & some integrations in Jest
+  - Current MFTF test cases pass (not additional ones)
+- Coding standards:
+  - Code does not violate best practices as defined (where?)
+  - Reviewed by X people representing Y stakeholders
```

Comment:
- Minimum number of reviewers for a major change should be 2
- Minimum number of reviewers for a minor change should be 1
- Minimum number of reviewers for a hotfix/patch should be 1, and that 1 must be the tech lead